src/you_get/extractors/embed.py

```diff
@@ -68,6 +76,17 @@ def embed_download(url, output_dir = '.', merge = True, info_only = False ,**kwa
         found = True
         iqiyi_download_by_vid((vid[1], vid[0]), title=title, output_dir=output_dir, merge=merge, info_only=info_only)
 
+    vids = matchall(content, qq_embed_patterns)
+    for vid in vids:
+        found = True
+        qq_download_by_vid(vid, title=title, output_dir=output_dir, merge=merge, info_only=info_only)
+
+    if len(matchall(content,letv_embed_patterns)):
```

Comment:
Although the return type of `matchall` can't be `None` (by definition), it would be better to make that bulletproof (just like all other matching patterns here).